.markdownlint.yml

```diff
@@ -0,0 +1,3 @@
+default: true
```

Comment:
could you move the config files to `tools/configs` ?

Reply:
Others like `.editorconfig`, `.prettierrc`, ``tslint.json` and many more are all tool configurations but not in the `tools/config` folder as well though 🤔😋

Reply:
There is a difference of daily used and cli tools (ng, npm,etc) 
and tools you may use once a week (and by npx something) ^^

I think all config files at root are mandatory right now, but if its moveable I'll try to move em too

Reply:
True that. Both tools have an option to define the config location so I will move it :+1: